Formula/apache-spark@2.4.6.rb

```diff
@@ -0,0 +1,29 @@
+class ApacheSparkAT246 < Formula
+  desc "Engine for large-scale data processing"
+  homepage "https://spark.apache.org/"
+  url "https://www.apache.org/dyn/closer.lua?path=spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz"
+  mirror "https://archive.apache.org/dist/spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz"
+  version "2.4.6"
+  sha256 "7174fee30057fbf226698ebfcfa7c2944a26197e149053aa29b26b250d1e1eba"
+  head "https://github.com/apache/spark.git"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
```

Comment:
Should now be `depends_on "openjdk@8"` (see https://github.com/eddies/homebrew-spark-tap/pull/3)